modules/apps/object/object-test/src/testFunctional/macros/json/JSONObject.macro

```diff
@@ -668,8 +668,15 @@ definition {
 			var userPassword = "test";
 		}
 
+		if (!(isSet(pageSize))) {
```

Comment:
here I'd rather play with `if isSet(pageSize)` and else for without the condition, so:

```
if (isSet(pageSize)) {	var url = "object-admin/v1.0/object-definitions?pageSize=${pageSize}";
		}
		else {
			var url = "object-admin/v1.0/object-definitions";
		}
```